apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/java/file/OperationBuilder.kt

```diff
@@ -244,5 +247,39 @@ internal class OperationBuilder(
         context.resolver.resolveOperationSelections(operation.name)
     )
   }
-}
 
+  private fun TypeSpec.Builder.addBuilder(context: JavaContext): TypeSpec.Builder {
+    addMethod(Builder.builderFactoryMethod())
+
+    val operationClassName = ClassName.get("", simpleName)
```

Comment:
JavaPoet certainly doesn't need to import the symbol in this file so it's certainly ok to pass `""` but since we have `packageName` available, this will make the code a bit more robust:

```suggestion
    val operationClassName = ClassName.get(packageName, simpleName)
```